src/lib/sampled.ts

```diff
@@ -127,19 +126,15 @@ export async function _sampledUrls(sitemapXml: string): Promise<string[]> {
       projDir += '/';
     }
 
-    const dirTreeRes = dirTree(projDir + 'src/routes');
-    routes = extractPaths(dirTreeRes);
-    // Match +page.svelte or +page@.svelte (used to break out of a layout).
-    //https://kit.svelte.dev/docs/advanced-routing#advanced-layouts-breaking-out-of-layouts
-    routes = routes.filter((route) => route.match(/\+page.*\.svelte$/));
+    routes = Object.keys(import.meta.glob('../../src/routes/**/+page*.svelte'));
 
     // 1. Trim everything to left of '/src/routes/' so it starts with
     //    `src/routes/` as `filterRoutes()` expects.
     // 2. Remove all grouping segments. i.e. those starting with '(' and ending
     //    with ')'
-    const i = routes[0].indexOf('/src/routes/');
+    const i = routes[0].indexOf('/routes/');
```

Comment:
This was necessary for the above change to pass tests, as the paths were relative and didn't include the `src` because it was a common parent - so like `../routes/dashboard/settings/+page.svelte`.